Resources/Locale/en-US/job/job-names.ftl

```diff
@@ -24,7 +24,8 @@ job-name-centcomoff = CentCom Official
 job-name-reporter = Reporter
 job-name-musician = Musician
 job-name-librarian = Librarian
-job-name-lawyer = Lawyer
```

Comment:
this should be commented out; google tells me .ftl uses HTML style comments. Not sure.

Reply:
also works ^

Reply:
I think having the full line-up of justice job names in one file is easier to understand. I've moved it to the DeltaV locale. Also, yaml comments seem to work just fine in this regard. I'm using them here since they're also used later in this file

Reply:
This is resolved?

Reply:
This is resolved